crates/flux-middle/src/rustc/lowering.rs

```diff
@@ -740,6 +740,21 @@ pub(crate) fn lower_ty<'tcx>(
             let args = lower_generic_args(tcx, args)?;
             Ok(Ty::mk_generator_witness(*did, args))
         }
+        rustc_ty::Dynamic(predicates, region, kind) => {
```

Comment:
We can emit unsupported here if we drop support for `dyn*`

Reply:
Done in `lower_dyn_kind`